flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/types/CRowSerializerTest.scala

```diff
@@ -29,6 +41,72 @@ class CRowSerializerTest extends TestLogger {
   @Test
   def testDefaultConstructor(): Unit = {
     new CRowSerializer.CRowSerializerConfigSnapshot()
+
+    val serializerConfigSnapshotClass = Class.forName(
+      "org.apache.flink.table.runtime.types.CRowSerializer$CRowSerializerConfigSnapshot")
```

Comment:
Can you use `classOf[...]` here?

Reply:
Thanks for the review. Have updated the PR to address your comments.